src/redux/store.js

```diff
@@ -0,0 +1,11 @@
+import { configureStore } from '@reduxjs/toolkit';
+import { bookSlice } from './books/booksSlice';
+import { categoriesSlice } from './categories/categoriesSlice';
+
+const store = configureStore({
+  reducer: {
+    Book: bookSlice.reducer,
+    categories: categoriesSlice.reducer,
```

Comment:
I have recorded [this video](https://www.loom.com/share/b444783841dd46768db2aa491f5f1da6) where I tried to explain what is wrong with this piece of code. I hope you find it helpful.